modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/dag/GenerateCodecs.scala

```diff
@@ -0,0 +1,20 @@
+package org.tessellation.rosetta.server.model.dag
+
+import scala.reflect.io.Path
+
+object GenerateCodecs {
+
+  def run(): Unit =
+    scala.tools.nsc.io.File
+      .apply(Path("modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model"))
+      .toDirectory
+      .list
+      .filter(_.isFile)
+      .foreach { p =>
+        val name = p.name.replaceAll(".scala", "")
+        val gen = s"""implicit lazy val ${name}Encoder = deriveConfiguredMagnoliaEncoder[${name}]
+                      |implicit lazy val ${name}Decoder = deriveConfiguredMagnoliaDecoder[${name}]""".stripMargin
+        println(gen)
+      }
```

Comment:
What is this?

Reply:
This is a work around for `sbt autogen` being broken (openAPI). @ryle-goehausen tried to get an autogen working, and this was the one that he found to be the least broken. I have added a `TODO` noting the purpose of this code and to remove it when feasible.

Reply:
Even if `sbt autogen` would be broken it wouldn't justify this code. Please implement this `TODO` as a part of the current changes.

Reply:
I will need assistance with this. We can go over it when we meet @tbekas next week. Additionally, please point me to a good tutorial.

Reply:
After discussing this with @ryle-goehausen , I'll be removing this file. I will work with @tbekas on Friday to automate this if we can do so relatively quickly. If not, we'll just leave this out of the code base.

Reply:
The generation of the `Rosetta` classes has ben made a custom `sbt` task. This is no longer needed.